### PR TITLE
ml-core-service-deployment-3.9.0

### PR DESCRIPTION
**Pull request by sivakarthikeyanoptit**
ml-core-service-deployment-3.9.0

### CODE REVIEW

**Comment by rjshrjndrn** [comment]
@sivakarthikeyanoptit Please resolve the conflicts

File: ansible/roles/stack-sunbird/templates/ml-core-service.env [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# ML core Service Config
+
+# Port on which the service will listen to
+APPLICATION_PORT={{ml_core_port|default("3000")}} 
+
+#Application ENV example local/dev/production
+APPLICATION_ENV={{ml_core_node_env|default("development")}}
+
+# Setting to turn on/off console logs
+ENABLE_CONSOLE_LOGGING={{ml_core_enable_debug_logging | default("ON")}}
```
Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/ml-core-service.env:10]: please disable debug log off by default, if in need, we'll override in private

File: kubernetes/helm_charts/core/ml-core-service/values.j2 [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+namespace: {{ namespace }}
+imagepullsecrets: {{ imagepullsecrets }}
+dockerhub: {{ dockerhub }}
+
+replicaCount: {{kendra_replicacount|default(1)}}
+repository: {{kendra_repository|default('ml-core-service')}}
+image_tag: {{ image_tag }}
+resources:
+  requests:
+    cpu: {{kendra_cpu_req|default('100m')}}
```
Comment (rjshrjndrn) [inline kubernetes/helm_charts/core/ml-core-service/values.j2:12]: Can you please rename the kendra variables?

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: kubernetes/helm_charts/core/ml-core-service/values.j2 (hunk 2 above)
Comment (sivakarthikeyanoptit) [reply kubernetes/helm_charts/core/ml-core-service/values.j2]: resloved

File: ansible/roles/stack-sunbird/templates/ml-core-service.env (hunk 1 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/stack-sunbird/templates/ml-core-service.env]: done

**Comment by sivakarthikeyanoptit** [comment]
merge conflicts has been resloved

**Comment by sivakarthikeyanoptit** [comment]
> @sivakarthikeyanoptit Please resolve the conflicts

merge conflicts has been resloved

**Review by rjshrjndrn — APPROVED** [review]

**Pull request merged**